### DISCUSSION TITLE
homebridge-nuvo might not be ready for Homebridge v2.0

### DISCUSSION THREAD

**Discussion by marsbars-marsbars** [Willmac16/homebridge-nuvo#22] [General]
Thank you for your work on the homebridge-nuvo plugin! I was wondering if you’re planning to update it for compatibility with Homebridge v2.0. Your efforts are greatly appreciated by the community!

**Comment by Willmac16** [comment]
I should be able to get this done when I come home from college this December.Is there anything else you’d like me to take a look at while I have access to my nuvo unit?Sent from my iPhoneOn Nov 18, 2024, at 08:57, marsbars-marsbars ***@***.***> wrote:﻿
Thank you for your work on the homebridge-nuvo plugin! I was wondering if you’re planning to update it for compatibility with Homebridge v2.0. Your efforts are greatly appreciated by the community!

**Reply by marsbars-marsbars** [reply]
Wonderful, thank you kindly.  I love the plugin, it keeps my Nuvo system modern. 

I have noticed from time to time, within the Apple home App. sometimes the volume is misrepresented.  Most of the time ~ 90% of the time its accurate.  However, where I've seen in inaccurate is when the volume is set at 1% and it shows at 100%   

Feature Request:   No means a priority, but it would be cool to be able to program my Nuvo Concerto audio zones from the your plug-in.  The only maintenance I use the old windows based Nuvo Software program for is to rename my audio zones when I switch something up.   https://legrand.webdamdb.com/directdownload.php?ti=55666903&tok=MVtFlStNfI9NWZSUNSJ14QRR

**Reply by Willmac16** [reply]
Looks like the current plugin supports Homebridge 2.0 out of the box.

The confusion at 100% has to due with (maybe older versions of) the home app issuing 100% brightness requests on zone power up. Without the override logic this causes speakers to run at full volume likely hurting ears.

I also worked through some of the power on volume issues, so you now should be able to request a percentage from the off state with only one command. Plus, turning the node on should respect the power on volume set in the plugin config.

**Comment by Willmac16** [comment]
I did write a command line interface a few years ago that comes along with the package for node configuration. You can’t be running homebridge at the same time, but it would let you rename the zones names that show on the keypads.On Nov 18, 2024, at 09:28, marsbars-marsbars ***@***.***> wrote:﻿
Wonderful, thank you kindly.  I love the plugin, it keeps my Nuvo system modern.
I have noticed from time to time, within the Apple home App. sometimes the volume is misrepresented.  Most of the time ~ 90% of the time its accurate.  However, where I've seen in inaccurate is when the volume is set at 1% and it shows at 100%
Feature Request:   No means a priority, but it would be cool to be able to program my Nuvo Concerto audio zones from the your plug-in.  The only maintenance I use the old windows based Nuvo Software program for is to rename my audio zones when I switch something up.   https://legrand.webdamdb.com/directdownload.php?ti=55666903&tok=MVtFlStNfI9NWZSUNSJ14QRR

**Reply by marsbars-marsbars** [reply]
Interesting - is it installed by default and are there some instructions I could test with ?  Also, I can wait until December.

**Reply by Willmac16** [reply]
Yup, should already be installed. I believe I’ve tested it on macOS and Linux, but might even work on windows

Instructions are on the [main readme](https://github.com/Willmac16/homebridge-nuvo?tab=readme-ov-file#nuvo-config-optional)

Let me know if you have questions. There should be a built in `-h` help flag or `help` command within the tool to ask the tool about its use and options.
